tokenline.c

```diff
@@ -23,13 +23,29 @@
 #define INDENT   "   "
 #define NO_HELP  "No help available."NL
 #define NL       "\r\n"
+#define CHARS	 "[]{}/\\_-!^.&%~" 
 
 static void line_clear(t_tokenline *tl);
 static void line_backspace(t_tokenline *tl);
 static void set_line(t_tokenline *tl, char *line);
 
 static char space[] = "               ";
 
+static void add_charz(t_tokenline *tl, int c)
```

Comment:
Function name is not self-explanatory.

Also, since it is similar to add_char(), should be placed just above or below the later.